### PR TITLE
Surface Previous Job Errors.

### PR DESCRIPTION
**Pull request by achettyiitr**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: warehouse/upload.go [hunk 1]
```diff
@@ -636,6 +636,17 @@ type TableUploadStatusT struct {
 	namespace     string
 	tableName     string
 	status        string
+	error         string
+}
+
+type TableUploadStatusErrorT struct {
```
Comment (psrikanth88) [inline warehouse/upload.go:642]: make it just TableUploadStatusT

Comment (achettyiitr) [inline warehouse/upload.go:642]: Update type names.

**Review by psrikanth88 — APPROVED** [review]

File: warehouse/upload.go (hunk 1 above)
Comment (achettyiitr) [reply warehouse/upload.go]: Done.

Comment (achettyiitr) [reply warehouse/upload.go]: Done.

**Review by psrikanth88 — APPROVED** [review]

**Pull request merged**